src/app/app.component.ts

```diff
@@ -281,16 +281,13 @@ export class AppComponent extends SubscriptionManager implements OnInit {
       if (component instanceof OverviewComponent) {
         component.locale = this.locale;
         this.showPageTitle = HeaderComponent.PAGE_TITLE_HIDDEN;
-        if (!this.header.countryTotalMap) {
-          this.loadLandingData(this.lastSetContentTierZeroValue);
-        }
       } else if (component instanceof CountryComponent) {
         this.countryComponentRef = component;
         component.includeCTZero = this.lastSetContentTierZeroValue;
         this.showPageTitle = HeaderComponent.PAGE_TITLE_MINIFIED;
-        if (!this.header.countryTotalMap) {
-          this.setCTZeroInputToLastSetValue(ctrlCTZero);
```

Comment:
It wasn't removed but moved down a level so it didn't need to be repeated!

(and this allows it to be picked up by the privacy policy / cookie policy pages)

Reply:
Hmm... this may be my lack of understanding. But the new line says this:
`this.loadLandingData(this.lastSetContentTierZeroValue);`
The old line says this: 
`this.setCTZeroInputToLastSetValue(ctrlCTZero);`
Question: is the old line contained in the new line?

Reply:
The new line calls `loadLandingData` with the `lastSetContentTierZeroValue`. 

That method will in turn set the variable in the landing page for ct-zero (after having loaded the correct data), and the control is set (for landing) in the `onOutletLoaded` handler.

Reply:
OK. As long as you considered this, all is well.